Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Tower/Staging/Subsequences/EnableCells.mo

```diff
@@ -0,0 +1,1238 @@
+﻿within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Tower.Staging.Subsequences;
+block EnableCells "Sequence for identifying enabing and disabling cells"
```

Comment:
This model needs revision to reflect sections 2. and 3 more simply and generally. Please come over to discuss if you'd like.